Added Frictionless Data

SimonScholler opened the pull request:
Frictionless data files have been added to the the the schema subfolder of the metadata folder. The data description tables in the Readme.md have been standardised and the frictionless schema is now linked in the Readme.md. For more information on the frictionless data format, see https://specs.frictionlessdata.io/ and https://framework.frictionlessdata.io/.

SimonScholler committed:
Adding frictionless data schema files for better machine-readability and data validation. Currently still work-in-progress.

SimonScholler committed:
Adding frictionless data schema files for better machine-readability and data validation. Currently still work-in-progress.

SimonScholler committed:
Adding frictionless data schema files for better machine-readability and data validation. Currently still work-in-progress.

SimonScholler committed:
Adding frictionless data schema files for better machine-readability and data validation. Currently still work-in-progress.

SimonScholler committed:
Adding frictionless data schema files for better machine-readability and data validation. Currently still work-in-progress.

SimonScholler committed:
Adding frictionless data schema files for better machine-readability and data validation. Currently still work-in-progress.

SimonScholler committed:
added valueDefinitions to frictionless data schema

SimonScholler committed:
Fine-tuning of frictionless data schema setup. Table specification in Readme updated / standardized. Some file cleanup.

SimonScholler committed:
Data table specification in Readme.md refined.

SimonScholler committed:
Frictionless Data package file moved from schemas to the metadata folder.

SimonScholler committed:
Refactoring of markdown data tables + changes in wording.

SimonScholler committed:
Readme.md: Data specification wording changed.

SimonScholler committed:
Frictionless package file renamed to 'datapackage.json' and moved to root folder and frictionless schemas to 'tableschema_XXX.json' (since these files are based on the Data Package Standard (https://datapackage.org/).

SimonScholler committed:
Minor text and formatting changes

HannesWuensche commented on Metadaten/schemas/tableschema_Corona_Datenspende_Erleben_und_Verhalten_Teildatensatz.json:
birthDate ist Datum YYYY
DATE ist Datum YYYY-MM-DD
ordinal Variablen und kategorische Variablen sind Integers nicht Strings
sessionTimestamp ist UNIX time > Datumsformat?

SimonScholler replied:
done, @HannesWuensche bitte nochmal reviewen und wenn alles passt mergen! (hab im schema jetzt auch 'valueDefinitions' durch das datapackage-äquivalent ersetzt ('categories'))

HannesWuensche replied:
userId, sessionTimestamp, sessionIteration sind noch ohne Beispiel. Richtig so?

SimonScholler replied:
Eigentlich ja, aber hab sie jetzt extra für dich noch hinzugefügt :)

SimonScholler committed:
changed:
birthDate ist Datum YYYY
DATE ist Datum YYYY-MM-DD
ordinal Variablen und kategorische Variablen sind Integers nicht Strings

SimonScholler committed:
schema fixed and valueDefinition properties renamed to be in line with datapackage standard

SimonScholler committed:
examples added to data specification

HannesWuensche committed:
Update header format
